ynr/apps/candidates/static/js/person_form.js

```diff
@@ -194,3 +195,16 @@ function makeTitleCase() {
     return title_case
   })
 }
+
+function addTitleCaseButton() {
+  let btn = document.createElement("button");
+  btn.innerHTML = "Title Case";
+  btn.type = "submit";
```

Comment:
If you change this from "submit" to "button" then you should stop that validation error / jump you see in the video. A submit button tells the browser to submit the form (causing validation errors), whereas a "button" does nothing unless it's hooked up (like you do below)